### PR TITLE
GH-19

### PR DESCRIPTION
**Pull request by dabhikrishna**

### CODE REVIEW

File: custom-plugin.php [hunk 1]
```diff
@@ -456,22 +472,28 @@ function my_plugin_page_content() {
...
+						echo '<td>' . esc_html( get_the_title( $post_id ) ) . '</td>';
+						echo '<td>' . esc_html( $client_name ) . '</td>';
+						echo '<td>' . esc_html( $address ) . '</td>';
+						echo '<td>' . esc_html( $email ) . '</td>';
+						echo '<td>' . esc_html( $phone ) . '</td>';
+						echo '<td>' . esc_html( $company ) . '</td>';
+						echo '<td>' . esc_html( get_the_date() ) . '</td>';
+						echo '<td><button class="delete-post-button" data-post-id="' . esc_html( $post_id ) . '" data-nonce="' . esc_attr( wp_create_nonce( 'delete_post_nonce' ) ) . '">Delete Post</button></td>';
 						echo '</tr>';
 					endwhile;
```
Comment (dharmendra-vadher) [inline custom-plugin.php:498]: move html code into template

File: templates/form.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      </style>';
+echo '<tr><th>Title</th>
+      <th>Name</th>
+      <th>Address</th>
+      <th>Email</th>
+      <th>Phone</th>
+      <th>Company Name</th>
+      <th>Date</th>
+      <th>Action</th>
+      </tr>';
```
Comment (dharmendra-vadher) [inline templates/form.php:19]: please close php tag and remove echo

File: templates/portfolio-renders.php
Comment (dharmendra-vadher) [inline templates/portfolio-renders.php:1]: why is this emtpy?

File: templates/portfolio-form.php
Comment (dharmendra-vadher) [inline templates/portfolio-form.php:1]: why is this empty

**Review by dharmendra-vadher — CHANGES_REQUESTED** [review]

File: templates/portfolio-renders.php
Comment (dabhikrishna) [reply templates/portfolio-renders.php:1]: not empty file here is html code

File: templates/portfolio-form.php
Comment (dabhikrishna) [reply templates/portfolio-form.php:1]: not empty file here is html code

**Pull request merged**